### PR TITLE
Fix dead link "root & docker" in Readme

### LINKED ISSUE #8943: Dead link in Readme
### Bug description

The link "root & docker" is dead:

image

### Steps to reproduce

View Readme

### Workspace affected

_No response_

### Expected behavior

_No response_

### Example repository

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by felladrin**
## Description

Fix dead link "root & docker" in Readme.

The second commit (intentionally left separate) applies formatting changes made by Prettier, so next time readme.md is edited, Prettier won't complain.

## Related Issue(s)

Fixes #8943

## How to test

- Open [readme.md](https://github.com/gitpod-io/gitpod/blob/vn/8943-dead-link-in-readme/README.md#features) from this branch.
- Confirm if the link "root & docker" is leading to the correct page.

## Release Notes

```release-note
NONE
```

## Documentation

None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +40,7 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
 
 👯‍♀️ [Collaboration](https://www.gitpod.io/docs/sharing-and-collaboration/) - invite team members to your dev environment or snapshot any state of your dev environment to share it with your team asynchronously.
 
-🛠 Professional & customizable developer experience - a Gitpod workspace gives you the same capabilities (yes, even [root & docker](https://www.gitpod.io/docs/feature-preview/#root-access)) as your Linux machine - pre-configured and optimized for your individual development workflow. Install any [VS Code extension](https://www.gitpod.io/docs/vscode-extensions/) with one click on a user and/or team level.
+🛠 Professional & customizable developer experience - a Gitpod workspace gives you the same capabilities (yes, even [root & docker](https://www.gitpod.io/docs/config-docker#configure-a-custom-dockerfile)) as your Linux machine - pre-configured and optimized for your individual development workflow. Install any [VS Code extension](https://www.gitpod.io/docs/vscode-extensions/) with one click on a user and/or team level.
```
Comment (gtsiolis) [inline README.md:43]: **praise**: Thanks for noticing, @felladrin! [FWIW](https://en.wiktionary.org/wiki/FWIW), the old link used to link to a dedicated page we had for inform users about an opt in to enable _feature preview_ that enabled multiple experimental features at once, see [relevant pull request](https://github.com/gitpod-io/retired-gatsby-website/pull/851).

**Review by gtsiolis — APPROVED** [review]
Thanks for taking a bias for action here, @felladrin! ✨ 

Builds failure seems unrelated. Let's merge this.

**Comment by geropl** [comment]
Sorry @felladrin @gtsiolis - this PR blocked the merge queue. I will re-trigger the build w/o preview env:

/werft run no-preview=true

**Pull request merged**